### DISCUSSION TITLE
Exception from 'wallet' {'error': "'NoneType' object has no attribute 'tx_store'", 'success': False}

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#3183] [General]
```
(venv) #screen#>chia farm summary
Exception from 'wallet' {'error': "'NoneType' object has no attribute 'tx_store'", 'success': False}
Farming status: Farming
```

This persists all the time, But farmer state is "farming"
Is it any serious? Can I deal with it by tweaking any config file?

upd: corresponding entry in the log:
```
22:33:02.925 wallet src.rpc.rpc_server         : WARNING  Error while handling message: Traceback (most recent call last):
  File "/home/xxxx/foreign_projects/chia-blockchain/src/rpc/rpc_server.py", line 81, in inner
    res_object = await f(request_data)
  File "/home/xxxx/foreign_projects/chia-blockchain/src/rpc/wallet_rpc_api.py", line 706, in get_farmed_amount
    tx_records: List[TransactionRecord] = await self.service.wallet_state_manager.tx_store.get_farming_rewards()
AttributeError: 'NoneType' object has no attribute 'tx_store'
```

upd 2:
```
(venv) #screen#>chia wallet get_address
No online backup file found, 
 Press S to skip restore from backup 
 Press F to use your own backup file: 
```

**Comment by chelseadole** [comment]
I am having this exact same issue. I tried `chia start farmer -r`, and the issue continued during `Farming status: Syncing` as well. Other than the exception, it _seems_ to be farming, but it concerns me. 

```
(venv) [root]# chia farm summary
Exception from 'wallet' {'error': "'NoneType' object has no attribute 'tx_store'", 'success': False}
Farming status: Farming
```

**Comment by BasilHorowt** [comment]
@chelseadole @ardabro I was having this issue to and found the solution in the [keybase chat](keybase://chat/chia_network.public#support/19739). Screenshot attached, hope it helps:


<img width="932" alt="Screen Shot 2021-04-09 at 8 10 46 PM" src="https://user-images.githubusercontent.com/82249557/114256661-b7436900-996f-11eb-9ef4-c42cc7621ee8.png">

**Reply by marramgg** [reply]
What happens here if you were running Chia on Windows and then start using it on Linux and experience this error? Will it have an impact on existing XCH (not that I have any but still)?

Thanks

**Comment by ardabro** [comment]
Anyway, I'd say that message is confusing. It should say something helpful connected with the context rather than a bare exception message. So I'm leaving it open.

**Comment by namelessperson0** [comment]
Also check https://github.com/Chia-Network/chia-blockchain/issues/1318 Seems it is not an issue.

**Comment by ardabro** [comment]
@namelessperson0 If scary, confusing, raw exception body presented to the user is not considered an issue I really don't understand today's standards programming standards :)

**Comment by namelessperson0** [comment]
> @namelessperson0 If scary, confusing, raw exception body presented to the user is not considered an issue I really don't understand today's standards programming standards :)

I agree. Peace ✌️.

**Comment by coinsccg** [comment]
@ardabro How did you solve this problem? I have the same problem

**Reply by mark-stopka** [reply]
Just run `docker exec -it chia venv/bin/chia wallet show` and hit S on the prompt.

**Reply by mdrodge** [reply]
Docker?

**Comment by mdrodge** [comment]
I have same issue using the GUI in windows.
I can't add new locations because of this error, in v1.1.5 i was able to just add locations to the config.yaml but since 1.1.6 that causes the app to not find plots and just give a spinny wheel on the plots page. still broken in 1.17
